lib/prm/repo.rb

```diff
@@ -194,8 +202,6 @@ def generate_release_gpg(path,release,gpg)
         Dir.chdir("#{path}/dists/#{release}") do
             if gpg.nil?
               sign_cmd = "gpg --yes --output Release.gpg -b Release"
-            elsif !gpg_passphrase.nil?
-              sign_cmd = "echo \'#{gpg_passphrase}\' | gpg -u #{gpg} --passphrase-fd 0 --yes --output Release.gpg -b Release"
```

Comment:
I had a similar problem with hashing algos, and so I pulled this patch down for my own use -- in testing it, I noticed that gpg_passphrase support no longer works. Was this removal intentional? (If it's got issues, I've no issues with possibly adjusting my workflow, just hoping to understand the context.)